### ISSUE TITLE
Process not exiting after running `gatsby build`

### ISSUE THREAD

**Issue by Tom-Bonnike** [gatsbyjs/gatsby#953]
I tried cloning gatsby-starter-default and it works properly, so IDK what could be wrong with my project. This is obviously annoying for tasks such as: `gatsby build --prefix-links && gh-pages -d public`
I simply fixed it by adding `process.exit()` when the command is done, but it’s still strange! Here’s the code if you want to take a look: https://github.com/Tom-Bonnike/portfolio-amine

EDIT: this twitter thread is interesting, I’ll investigate on my own project when I’ll have the time :)
https://twitter.com/kentcdodds/status/863070381648859136

**Comment by Tom-Bonnike** [comment]
https://github.com/gatsbyjs/gatsby/compare/master...Tom-Bonnike:master

**Comment by Tom-Bonnike** [comment]
Oh, I found the culprit. A setTimeout in GSAP’s TweenLite. I’ll see what I can do on their side :)
If someone ever has the same problem, here’s how I debugged it:
I removed every page, one by one, until I was only left with my _template.js only rendering a `<div>` and my JSON wrapper. I was actually rendering a component in this wrapper, that was using TweenLite.
I made it so that the component would only render a `<div>` and then commented out every dependency one by one until I found out it was TweenLite.
As suggested on the twitter thread that I linked, I used https://github.com/mafintosh/why-is-node-running to see what was making node hang in TweenLite :)
